subprojects/workers/workers.gradle

```diff
@@ -0,0 +1,11 @@
+
+sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
```

Comment:
Can be `1.6` I think. It should really be `1.6` for the bits that run in the worker process and `1.7` for the bits that run on the build process, but we can leave that for later.